src/main/java/org/folio/search/utils/SearchUtils.java

```diff
@@ -307,6 +307,16 @@ public static boolean isEmptyString(Object value) {
     return value instanceof String string && string.isEmpty();
   }
 
+  /**
+   * Check if passed value is blank.
+   *
+   * @param value - value to check
+   * @return true - if value is blank, false - otherwise
+   */
+  private static boolean isBlankString(String value) {
```

Comment:
I don't think that this method makes sense, as there no other logic apart from using string utils method